odl/set/pspace.py

```diff
@@ -256,15 +258,16 @@ def element(self, inp=None):
         if inp is None:
             inp = [space.element() for space in self.spaces]
 
-        # TODO: how does this differ from "if inp in self"?
```

Comment:
Must have been me. Yes, I remember: doesn't `__contains__` check for the same? (without looking it up)